wasm/vm_num_test.go

```diff
@@ -1,3 +1,40 @@
 package wasm
 
-// fixme
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func buildvm() *VirtualMachine {
+	return &VirtualMachine{
+		OperandStack: NewVirtualMachineOperandStack(),
+	}
+}
+
+func Test_i32eqz(t *testing.T) {
+	vm := buildvm()
+	vm.OperandStack.Push(uint64(0))
+	i32eqz(vm)
+	assert.Equal(t, uint32(1), uint32(vm.OperandStack.Pop()))
+}
+
+func Test_i32eq(t *testing.T) {
+	vm := buildvm()
+	vm.OperandStack.Push(uint64(3))
+	vm.OperandStack.Push(uint64(3))
+	i32eq(vm)
+	assert.Equal(t, uint32(1), uint32(vm.OperandStack.Pop()))
+}
+
+func Test_i32ne(t *testing.T) {
+	vm := buildvm()
```

Comment:
can you rewrite tests with multiple cases in the Table Drive style? https://github.com/golang/go/wiki/TableDrivenTests